torchaudio/datasets/tedlium.py

```diff
@@ -0,0 +1,116 @@
+import os
+from typing import Tuple
+
+import torchaudio
+from torch import Tensor
+from torch.utils.data import Dataset
+from torchaudio.datasets.utils import (
+    download_url,
+    extract_archive,
+    walk_files,
+)
+
+RELEASE = "release1"  # Default release
+
+_RELEASE_CONFIGS = {
```

Comment:
I just ran `sha256sum` and got different values. Which algorithm did you use?

```
$ sha256sum archives/tedlium/*
ad1e454d14d1ad550bc2564c462d87c7a7ec83d4dc2b9210f22ab4973b9eccdb  archives/tedlium/TEDLIUM_release-3.tgz
30301975fd8c5cac4040c261c0852f57cfa8adbbad2ce78e77e4986957445f27  archives/tedlium/TEDLIUM_release1.tar.gz
93281b5fcaaae5c88671c9d000b443cb3c7ea3499ad12010b3934ca41a7b9c58  archives/tedlium/TEDLIUM_release2.tar.gz
```

Reply:
My bad here, I used sha1sum by mistake, I runned again and got your same hashes, will update them in the next commit